dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/validation/hooks/PreCheckSecurityOwnershipValidationHook.java

```diff
@@ -122,16 +124,24 @@ public void validateTrackedEntity( ValidationErrorReporter reporter, TrackedEnti
             if ( tei.getProgramInstances().stream().anyMatch( pi -> !pi.isDeleted() )
                 && !user.isAuthorized( Authorities.F_TEI_CASCADE_DELETE.getAuthority() ) )
             {
-                addError( reporter, E1100, user, tei );
+                TrackerErrorReport error = TrackerErrorReport.builder()
```

Comment:
I don't recall why. It might be a leftover from an experiment with how to refactor. I will revisit adding errors later on again 👍🏻 I also got some comments, suggestions from Jan on this 😄